utils/prof_utils.py

```diff
@@ -0,0 +1,819 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import re
+import sys
+import logging
+import argparse
+from scipy.interpolate import UnivariateSpline
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+from astropy.time import Time
+from scipy.optimize import curve_fit
+from scipy.stats import chisquare
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused chisquare imported from scipy.stats](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4645369)